modules/ROOT/pages/dataweave-troubleshoot.adoc

```diff
@@ -203,3 +207,18 @@ Trace:
 ----
 
 Making the output explicit solves the issue: `output application/java --- payload splitBy ' '`
+
+== DataWeave Warnings
+
+The following are some common DataWeave warnings that you can find in your dump file, along with some details to address these warnings.
+
+=== End of Input Was Reached
+
+This warning shows because DataWeave treats the Unicode non-character `U+FFFF` as a special character that indicates the end of input.
```

Comment:
```suggestion
This warning appears because DataWeave treats the Unicode non-character `U+FFFF` as a special character that indicates the end of input.
```